### ISSUE TITLE
alt text - GOV.UK change of guidance

### ISSUE THREAD

**Issue by sarawilcox** [nhsuk/nhsuk-service-manual-backlog#287]
## What
GOV.UK published new advice on alt text in September. We need to understand why the guidance has changed in order to decide whether or not it's appropriate for NHS.UK.

### "If you already have descriptive alt text 

If you think the alternative text on your image is already descriptive enough and adequately serves the same purpose as the image, that is what your content needs to meet WCAG 2.1. You do not need to make changes before 23 September.
You should remove the alt text and move it to the body content after the deadline or when you have some more time . 

This will ensure that:

- the descriptions are available to all users, for example people using magnification or custom colour schemes

- all images are well described

Apologies we didn’t make this clearer in the original post."

## Original post (4 September):
"The message below was sent to all the GOV.UK leads yesterday and posting here to make sure everyone is aware:
GOV.UK needs to meet the deadline for the accessibility regulations, which is 23 September. Departments are responsible for making sure their own content is accessible.

As part of the work to get GOV.UK ready for the deadline, we’ve:

- identified all the images on GOV.UK 

- updated content design guidance to make images as accessible as possible 

There are 2 kinds of images:
1. Decorative images, for example a photo of a person or team.
2. Images which convey useful information, like diagrams and graphs.

What you need to do: 

You need to check that images in your content that convey useful information have good descriptions. This means that the images are fully described to people that cannot see them. Decorative images do not need describing.

Find your department’s images by filtering column E of the attached spreadsheet. There are .csv and .ods versions attached to this email. 

Check you’ve described each image that contains useful information in the body text. Leave the alt text field for all images empty. Use Content design: Images and How to publish on GOV.UK, Images and videos to help. 

Let us know when you’ve finished by sending a confirmation to GOV.UK Accessibility updates - images. 
If you have any questions, please reply to this email. "

**Comment by sarawilcox** [comment]
Related to [image component](https://github.com/nhsuk/nhsuk-service-manual-backlog/issues/28)

**Comment by sarawilcox** [comment]
@PaulineRichardson is going to follow this ticket up.

**Comment by PaulineRichardson** [comment]
This has been discussed in Accessibility meeting this morning regarding images and use of captions and alt text.

It was agreed that the images displayed on GOV.UK are more likely to be there for aesthetic purposes as opposed to supplementing written health content on NHS.UK. As such, we will keep our alt text descriptions placement as they are currently.

**Comment by sarawilcox** [comment]
Thanks @PaulineRichardson. 
Should we keep this open pending further consideration? Also pending work on [skin symptoms](https://github.com/nhsuk/nhsuk-service-manual-backlog/issues/181)?
Should we rename this issue "alt text" now, given that we're not planning on follow GOV.UK advice?

**Comment by PaulineRichardson** [comment]
Hi @sarawilcox, yes I think would make sense to keep open to further consideration. I have changed the title to 'alt text.'

**Comment by Tosin-Balogun** [comment]
This was **ranked 4th**  from the `proposed` group at the content backlog prioritisation workshop held on 24-Nov-2020

**Comment by PaulineRichardson** [comment]
Thanks @Tosin-Balogun

**Comment by PaulineRichardson** [comment]
We spoke about it this in the Accesibility awareness group and no issues were reported. We felt that .GOV do deliver image content differently to us and any breeches in AA accessibility on our images would have surfaced from our Accessibility Audit with Nomensa. We are therefore satisfied that we are reaching AA standard with our current alt-text solution however we are seeking clarification from Alistair Duggin and Alastair Campbell at Nomensa to confirm our position is correct.

**Comment by PaulineRichardson** [comment]
Responses from Nomensa: 

'From an accessibility regulations, and being compliant with WCAG, point of view there is not a requirement to move the text alternative from the image alt text to the body content.  The recommendation is for the benefit of people with low vision (who are not using a screen reader) who may benefit from being able to read a text description of the image – but is not required to meet WCAG.'

'Experience from the previous project, I don’t think it would help to move the alt text into the main content.
 
There might be some instances where it would help, but in your example (chickenpox) the pictures are zoomed in, and the captions already provide some of that information.
 
Adding the alt text above or below could be confusing for the general population and people with cognitive impairments.
 
I suspect the advice comes from looking at different scenarios from your conditions pages, and doesn’t make sense in this context.
 
(They also didn’t seem to make clear that the images would need null alt text, but maybe that was further down.)
 
The pages and pictures zoom effectively, so I think the low-vision case is well covered.'

**Comment by sarawilcox** [comment]
Sorry, I'm going to keep it open but put it in the Done column.

I wonder whether it's worth posting something in Slack just in case people wondered whether we should follow GOV.UK. Would you be able to do that @PaulineRichardson?

**Comment by sarawilcox** [comment]
Our current guidance - unchanged: https://service-manual.nhs.uk/accessibility/content#use-alternative-text-for-images-in-content
